apps/system/utils/Kconfig

```diff
@@ -53,7 +53,7 @@ endif #NET_CMDS
 config ENABLE_CPULOAD_CMD
 	bool "cpuload"
 	default y
-	depends on SCHED_CPULOAD
+	depends on SCHED_CPULOAD && FS_PROCFS && !_FS_PROCFS_EXCLUDE_CPULOAD
```

Comment:
1. Please remove `_` at start of  `_FS_PROCFS_EXCLUDE_PROCESS`.
2. Is there a way to enable `FS_PROCFS_EXCLUDE_PROCESS` without enabling `FS_PROCFS`?
   If there is not, let's remove `FS_PROCFS` from dependancy. It's a redundant.